CHANGELOG.md

```diff
@@ -4,10 +4,18 @@
 
 ### Features
 
-- **auth**: Fix cancelling hostedUI returning a generic error (#982)
 - Support lazy load and pagination for API (#1009)
 - Separate DataStore List logic out to list provider (#1000)
 
+### ⚠ BREAKING CHANGES
+
+- **auth**: Fix cancelling hostedUI returning a generic error (#982). User cancelled error returned from `Amplify.Auth.signInWithWebUI` is changed from `SFAuthenticationError.canceledLogin` to `AuthError.service(_, _, AWSCognitoAuthError.userCancelled)`.
+
+
+### Misc updates
+
+- **auth**: `Amplify.Auth.signInWithWebUI` uses `ASWebAuthenticationSession` internally for iOS 13.0+. For older iOS versions, it will fallback to `SFAuthenticationSession`. You can also make use of `ASWebAuthenticationSession`'s `prefersEphermeralWebBrowserSession` by setting the value `preferPrivateSession` of `AWSAuthWebUISignInOptions` during signIn. If `preferPrivateSession` was set during signIn, the user will not see a `ASWebAuthenticationSession` popup during signOut. 
```

Comment:
This should be moved back under Features. Suggested reword:

```suggestion
- **Auth**: `Amplify.Auth.signInWithWebUI` now automatically uses `ASWebAuthenticationSession` internally for iOS 13.0+. For older iOS versions, it will fall back to `SFAuthenticationSession`.

    This release also introduces a new `preferPrivateSession` flag to `AWSAuthWebUISignInOptions` during the sign in flow. If `preferPrivateSession` is set to `true` during sign in, and the user's preferred browser supports [`ASWebAuthenticationSession.prefersEphemeralWebBrowserSession`](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession/3237231-prefersephemeralwebbrowsersessio), the user will not see a web view displayed when they sign out. 
    
    <Add usage example with convenience static method in ` ` `swift block>
```

Reply:
Added